numpy/core/_add_newdocs.py

```diff
@@ -3336,6 +3336,41 @@
     """)
 
 
+add_newdoc('numpy.lib.utils', 'byte_bounds',
```

Comment:
This isn't the right approach.

Functions defined in python should use regular docstrings - `_add_newdocs` is a hack to make adding docstrings to C function easier. It should not be used for python functions.

Reply:
@eric-wieser that means I need to only include docstrings of these functions? However, there are `_add_newdocs` used for functions like `may_share_memory` as well which are python functions defined in `multiarray.py`, could you explain why these two cases are different? Thanks.

Reply:
Good catch, those docstrings should also not be there. Fixed in gh-13876.